pytorch_lightning/plugins/environments/lightning_environment.py

```diff
@@ -96,7 +96,6 @@ def find_free_network_port() -> int:
     """
     s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
     s.bind(("", 0))
-    s.listen(1)
```

Comment:
So this is redundant because we don't want to actually listen here but just create the socket to find a free port?

Reply:
Yes, I'm basing this around the answer here https://stackoverflow.com/questions/1365265/on-localhost-how-do-i-pick-a-free-port-number
I saw the listen(1) call in one example but couldn't figure out why it is there.

Reply:
as far as I understood this is only necessary if this socket is actually going to receive some info which is nothing we handle there :)